# Generated by Django 4.1.1 on 2022-11-27 17:52

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='MassMailErrors',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('email', models.CharField(blank=True, max_length=255, null=True, verbose_name='Email')),
                ('date', models.DateTimeField(auto_now=True, null=True, verbose_name='Дата и время ошибки')),
                ('text', models.TextField(blank=True, null=True, verbose_name='Текст ошибки')),
            ],
            options={
                'verbose_name': 'Ошибка при массовой рассылке',
                'verbose_name_plural': 'Ошибки при массовой рассылке',
            },
        ),
        migrations.CreateModel(
            name='MassMailSuccess',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('number', models.PositiveIntegerField(blank=True, default=0, null=True, verbose_name='Отправлено сообщений')),
                ('name', models.CharField(blank=True, max_length=255, null=True, verbose_name='Имя')),
            ],
            options={
                'verbose_name': 'Результат массовой рассылки',
                'verbose_name_plural': 'Результаты массовой рассылки',
            },
        ),
        migrations.CreateModel(
            name='OrderMailErrors',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(blank=True, max_length=255, null=True, verbose_name='Имя')),
                ('email', models.CharField(blank=True, max_length=255, null=True, verbose_name='Email')),
                ('date', models.DateTimeField(auto_now=True, null=True, verbose_name='Дата и время ошибки')),
                ('text', models.TextField(blank=True, null=True, verbose_name='Текст ошибки')),
            ],
            options={
                'verbose_name': 'Ошибка оповещения о заказе',
                'verbose_name_plural': 'Ошибки оповещений о заказах',
            },
        ),
        migrations.CreateModel(
            name='OrderSaveErrors',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(blank=True, max_length=255, null=True, verbose_name='Имя')),
                ('date', models.DateTimeField(auto_now=True, null=True, verbose_name='Дата и время ошибки')),
                ('text', models.TextField(blank=True, null=True, verbose_name='Текст ошибки')),
            ],
            options={
                'verbose_name': 'Ошибка сохранения заказа',
                'verbose_name_plural': 'Ошибки сохранения заказов',
            },
        ),
    ]
